packages/block-editor/src/components/rich-text/content.scss

```diff
@@ -10,6 +10,8 @@
 		// Use opacity to work in various editor styles.
 		// We don't specify the color here, because blocks or editor styles might provide their own.
 		opacity: 0.62;
+		// Reset the text shadow from the global styles.
+		text-shadow: none;
```

Comment:
I wasn't sure whether to add this, but if we don't reset the shadow, the placeholder is very hard to see.

<img width="275" height="97" alt="Image" src="https://github.com/user-attachments/assets/bfd8b4be-4f9e-4f3c-a414-ef74f2943362" />